data-prepper-core/src/test/resources/valid_multiple_processors.yml

```diff
@@ -16,7 +16,7 @@ raw-pipeline:
     - string_converter:
         upper_case: false
   sink:
-    - file:
+    - stdout:
```

Comment:
Is the concern here that file writer is open on pipeline construction but will not be closed properly in tests?